common/src/androidTest/java/com/microsoft/identity/common/SharedPreferencesFileManagerTests.java

```diff
@@ -101,65 +102,61 @@ public void testGetString() {
 
     @Test
     public void testGetSharedPreferences() throws Exception {
-        Field f = SharedPreferencesFileManager.class.getDeclaredField("mStorageHelper");
+        Field f = SharedPreferencesFileManager.class.getDeclaredField("mEncryptionManager");
         f.setAccessible(true);
 
+        IKeyAccessor keyAccessor = (null == f.get(mSharedPreferencesFileManager))? null : sTEST_ENCRYPTION_MANAGER;
+
         Assert.assertSame(mSharedPreferencesFileManager, SharedPreferencesFileManager.getSharedPreferences(
-                InstrumentationRegistry.getTargetContext(),
+                ApplicationProvider.getApplicationContext(),
                 mSharedPreferencesFileManager.getSharedPreferencesFileName(),
-                (IStorageHelper) f.get(mSharedPreferencesFileManager)
+                keyAccessor
         ));
     }
 
     @Test
-    public void testGetSharedPreferencesNoStorageHelper() throws Exception {
-        Field f = SharedPreferencesFileManager.class.getDeclaredField("mStorageHelper");
+    public void testGetSharedPreferencesNoKeyAccessor() throws Exception {
+        Field f = SharedPreferencesFileManager.class.getDeclaredField("mEncryptionManager");
         f.setAccessible(true);
-        IStorageHelper storageHelper = (IStorageHelper) f.get(mSharedPreferencesFileManager);
-        IStorageHelper newStorageHelper;
-        if (storageHelper == null) {
-            newStorageHelper = new StorageHelper(InstrumentationRegistry.getTargetContext());
+        IKeyAccessor mkeyAccessor = (null == f.get(mSharedPreferencesFileManager))? null : sTEST_ENCRYPTION_MANAGER;
+
+        IKeyAccessor keyAccessor = mkeyAccessor;
+        IKeyAccessor newKeyAccessor;
+        if (keyAccessor == null) {
+            newKeyAccessor = new AndroidAuthSdkStorageEncryptionManager(ApplicationProvider.getApplicationContext(), null);
         } else {
-            newStorageHelper = null;
+            newKeyAccessor = null;
         }
         Assert.assertNotSame(mSharedPreferencesFileManager, SharedPreferencesFileManager.getSharedPreferences(
-                InstrumentationRegistry.getTargetContext(),
+                ApplicationProvider.getApplicationContext(),
                 mSharedPreferencesFileManager.getSharedPreferencesFileName(),
-                newStorageHelper)
+                newKeyAccessor)
         );
         Assert.assertSame(mSharedPreferencesFileManager, SharedPreferencesFileManager.getSharedPreferences(
-                InstrumentationRegistry.getTargetContext(),
+                ApplicationProvider.getApplicationContext(),
                 mSharedPreferencesFileManager.getSharedPreferencesFileName(),
-                storageHelper)
+                keyAccessor)
         );
     }
 
-    @Test
-    public void testGetSharedPreferencesWithAndWithoutStorageHelper() throws Exception {
```

Comment:
bring this back?

Reply:
removed it because it will fail.
running with first parameter when the encryptionManager = null 
mSharedPreferencesFileManager is equal to SharedPreferencesFileManager.getSharedPreferences(InstrumentationRegistry.getTargetContext(),mSharedPreferencesFileManager.getSharedPreferencesFileName(),null)
Anyway this is tested when running testGetSharedPreferences  with first parameter

Reply:
I think at one point, the system always returned a new one when encryption manager was null.  Very strange.